cmd/runsim/main.go

```diff
@@ -37,135 +38,141 @@ var (
 		977367484, 491163361, 424254581, 673398983,
 	}
 	seedOverrideList = ""
+	results          chan *Seed
 
 	// goroutine-safe process map
 	procs map[int]*os.Process
 	mutex *sync.Mutex
 
-	// results channel
-	results chan bool
-
 	// command line arguments and options
-	jobs         = runtime.GOMAXPROCS(0)
-	pkgName      string
-	blocks       string
-	period       string
-	testname     string
-	genesis      string
-	exitOnFail   bool
-	githubConfig string
-	logObjprefix string
-	slackConfig  string
-
-	// integration with Slack and Github
-	slackToken   string
-	slackChannel string
-	slackThread  string
-
-	// logs temporary directory
-	tempdir string
+	jobs                                  = runtime.GOMAXPROCS(0)
+	pkgName                               = "./simapp"
+	genesis, blocks, period, testname     string
+	simId, hostId                         string
+	notifySlack, notifyGithub, exitOnFail bool
+
+	// log stuff
+	logObjPrefix  string
+	runsimLogFile *os.File
 )
 
+type Seed struct {
+	Num          int
+	Stdout       string
+	Stderr       string
+	ExportParams string
+	ExportState  string
+	Failed       bool
+}
+
 func init() {
 	log.SetPrefix("")
 	log.SetFlags(0)
 
 	procs = map[int]*os.Process{}
 	mutex = &sync.Mutex{}
 
-	flag.IntVar(&jobs, "j", jobs, "Number of parallel processes")
-	flag.StringVar(&genesis, "g", "", "Genesis file")
-	flag.StringVar(&seedOverrideList, "seeds", "", "run the supplied comma-separated list of seeds instead of defaults")
-	flag.BoolVar(&exitOnFail, "e", false, "Exit on fail during multi-sim, print error")
-	flag.StringVar(&logObjprefix, "log", "", "S3 object key for log files")
-	flag.StringVar(&githubConfig, "github", "", "Report results to Github's PR")
-	flag.StringVar(&slackConfig, "slack", "", "Report results to slack channel")
+	flag.StringVar(&genesis, "Genesis", "", "genesis file path")
```

Comment:
I'd define the variables for the flags in `flag.go` for context awareness. Create a `function initializeFlags()` which is called on `init()`